app/serializers/viewer_member_info_serializer.rb

```diff
@@ -40,7 +40,7 @@ def as_hash
   def included_members
     @included_members ||= begin
       members = work.members.order(:position)
-      members = members.where(published: true) if current_user.nil?
+      members = members.where(published: true) unless @show_unpublished
```

Comment:
we have a method `show_unpublished`, so should access that instead of the instance variable. Our "style guide" is kind of never to access instance variables directly outside of setting them in intializer, caues it's easy to make a typo that doesn't get caught and other reasons.

Reply:
Fixed.